packages/blockchain-extension/extension/dependencies/DependencyManager.ts

```diff
@@ -530,16 +530,36 @@ export class DependencyManager {
                 if (!remote && semver.lt(info.longVersion, '6.0.0')) {
                     progress.report({ message: `Updating ${dependency}` });
                     outputAdapter.log(LogType.INFO, undefined, `Updating ${dependency}`);
-                    const basePath: string = path.join(extensionPath, 'node_modules', 'grpc', 'src', 'node', 'extension_binary');
+                    let basePath: string = path.join(extensionPath, 'node_modules', 'grpc', 'src', 'node', 'extension_binary');
 
-                    const origPath: string = path.join(basePath, `node-v${info.modules}-${os}-${architecture}-${thing}`);
-                    const newPath: string = path.join(basePath, `electron-v${info.shortVersion}-${os}-${architecture}-${thing}`);
+                    let origPath: string = path.join(basePath, `node-v${info.modules}-${os}-${architecture}-${thing}`);
+                    let newPath: string = path.join(basePath, `electron-v${info.shortVersion}-${os}-${architecture}-${thing}`);
 
-                    const exists: boolean = await fs.pathExists(origPath);
+                    let exists: boolean = await fs.pathExists(origPath);
                     if (exists) {
                         await fs.remove(origPath);
                     }
                     await fs.rename(newPath, origPath);
+
+                    // this is probably only needed in development
+                    const otherGRPC: string[] = ['ibm-blockchain-platform-gateway-v1'];
+
+                    for (const other of otherGRPC) {
+                        basePath = path.join(extensionPath, 'node_modules', `${other}`, 'node_modules', 'grpc', 'src', 'node', 'extension_binary');
+
+                        origPath = path.join(basePath, `node-v${info.modules}-${os}-${architecture}-${thing}`);
+                        newPath = path.join(basePath, `electron-v${info.shortVersion}-${os}-${architecture}-${thing}`);
+
+                        exists = await fs.pathExists(origPath);
+                        if (exists) {
+                            await fs.remove(origPath);
```

Comment:
Is it worth try/catching this, just in case?

Reply:
it will get caught be the failed to activate extension bit. The same as the other part that checks if the files exist in the main extension bit. The new code will only ever happen in development, the files will never exist in prod